learnCSharp/Commands.cs

```diff
@@ -11,68 +12,38 @@ public static class Commands
         
         public static void ListAllCommands()
         {
-            Console.WriteLine("new: create a new todo task.(can enter in order: \'new\' Description");
-            Console.WriteLine("task: enter task description. (can enter in order: \'task\' OldDescription,NewDescription");
-            Console.WriteLine("status: Set isDone (true or false). (can enter in order: \'status\' Description,NewStatus");
-            Console.WriteLine("print: print selected task description, status, and ID. (can enter as: \'print\' Description");
-            Console.WriteLine("remove: remove chosen task. (can enter as: \'remove\' Description");
+            Console.WriteLine("new: create a new todo task.(can enter in order: \'new\' description");
+            Console.WriteLine("task: enter task description. (can enter in order: \'task\' idNum,NewDescription");
+            Console.WriteLine("status: Set isDone (true or false). (can enter in order: \'status\' idNum,NewStatus");
+            Console.WriteLine("print: print selected task description, status, and ID. (can enter as: \'print\' idNum");
+            Console.WriteLine("remove: remove chosen task. (can enter as: \'remove\' idNum");
             Console.WriteLine("list: print all tasks.\nowner: change current user. (can enter as \'owner\' User" +
                 "\nexit: exit loop.");
         }
 
         
-        public static void PrintAllTodos(List<Todo> list)
+        public async static void PrintAllTodos(ITodoRepository todoRepository)
         {
-            foreach (Todo a in list)
+            foreach (Todo a in await todoRepository.GetAll())
             {
                 a.ToString();
             }
         }
 
         
-        public static void RemoveTodo(List<Todo> list, String[] input)
+        public async static void RemoveTodo(ITodoRepository todoRepository, String[] input)
         {
             var listIndex = -1;
             var indexFound = false;
             if (input.Length > 1)
             {
-                foreach (Todo a in list)
-                {
-                    if (a.descript.Equals(input[1]))
-                    {
-                        listIndex = list.IndexOf(a);
-                        indexFound = true; break;
-                    }
-                }
-                if (indexFound)
-                {
-                    list.Remove(list[listIndex]);
-                }
-                else
-                {
-                    Console.WriteLine("Task with input description not found.");
-                }
+                await todoRepository.DeleteTodo(int.Parse(input[1]));
             }
             else
             {
                 Console.WriteLine("ENTER CURRENT DESCRIPTION OF TASK");
```

Comment:
It looks like we switched this to use Id instead of Description; the text should probably be updated to reflect that.